_includes/payment-order-other-features.md

```diff
@@ -142,6 +142,13 @@ Common sub-resources like [payeeinfo][payee-reference], that are
 structurally identical for both payments and payments orders, are described in
 the [Payment Resources][payment-resource] section.
 
+## Instrument Mode
+
+You will have the possibility to use the `PaymentOrder` API but instead of 
+getting a menu with all instrument setup on your account, you will be able to 
+decide for each payment which instrument to show. 
+This is possible by using the request field `instrument`.
```

Comment:
```suggestion
In "Instrument Mode" the Payment Menu will display only one specific payment
instrument instead of all configured on your merchant account. The Payment Order
resource works just like it otherwise would, allowing you to remain largely
indifferent to the payment instrument in use.

The Payment Menu is switched to "Instrument Mode" by providing the request field
`instrument` as described below.
```